### PR TITLE
Fix broken docker link

### PR DESCRIPTION
**Pull request by ChrsMark**
Fixes #

## Changes

The link `https://docs.docker.com/engine/reference/run/#container-identification` is outdated. This makes the `markdown-link-check` to fail.

Spotted at https://github.com/open-telemetry/semantic-conventions/actions/runs/10383411357/job/28748273043?pr=1251.

## Merge requirement checklist

* [x] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [x] Change log entry added, according to the guidelines in [When to add a changelog entry](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#when-to-add-a-changelog-entry).
  * If your PR does not need a change log, start the PR title with `[chore]`
* [x] [schema-next.yaml](https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml) updated with changes to existing conventions.

### CODE REVIEW

**Comment by ChrsMark** [comment]
Hmm, it seems that docker's doc pages are failing here there actually 🤔 .

**Comment by jsuereth** [comment]
Sometimes websites prevent bots from hitting them.  We may want to remove dockerhub as a validated location in the link checker...

**Comment by lmolkova** [comment]
I think it'll self-heal - https://github.com/docker/docs/pull/20606

**Comment by lmolkova** [comment]
> I think it'll self-heal - [docker/docs#20606](https://github.com/docker/docs/pull/20606)

self-healed!

File: .chloggen/fix_docker_link.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the area of concern in the attributes-registry, (e.g. http, cloud, db)
+component: container
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Fixes broken link
```
Comment (lmolkova) [inline .chloggen/fix_docker_link.yaml:13]: no need to add changelog for minor changes like this (you can add [chore] in title to disable the check)

**Review by lmolkova — APPROVED** [review]

**Review by jsuereth — APPROVED** [review]

**Pull request merged**